### PR TITLE
Increase default TNF to v5.2.1

### PR DESCRIPTION
**Pull request by tonyskapunk**
Include default vars in the tnf feedback for new tests introduced for operators.

---

Test-hints: no-check

### CODE REVIEW

File: roles/cnf_cert/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ tnf\_stage                              | "tests"
...
-test\_network\_function\_version        | v5.2.0                                                                                                                                                                                                                                 | CNF Cert Suite version downloaded. DCI App Agent supports from v4.4.0 till the latest stable version. HEAD version (in the main branch) can be also used, but it is not guaranteed a complete compatibility with the latest unstable changes.
+test\_network\_function\_version        | v5.2.1                                                                                                                                                                                                                                 | CNF Cert Suite version downloaded. DCI App Agent supports from v4.4.0 till the latest stable version. HEAD version (in the main branch) can be also used, but it is not guaranteed a complete compatibility with the latest unstable changes.
```
Comment (betoredhat) [inline roles/cnf_cert/README.md:15]:
```suggestion
test\_network\_function\_version        | See roles/cnf_cert/defaults/main.yaml for the latest version supported by the agent                                                                                                                                                                                                                           | CNF Cert Suite version downloaded. DCI App Agent supports from v4.4.0 till the latest stable version. HEAD version (in the main branch) can be also used, but it is not guaranteed a complete compatibility with the latest unstable changes.
```

**Review by betoredhat — APPROVED** [review]
LGTM :100:

**Review by betoredhat — APPROVED** [review]
LGTM

**Comment by ramperher** [comment]
Thanks for checking this @tonyskapunk , you did exactly what was needed for this new release :)

**Pull request merged**